ja/news/_posts/2016-12-25-ruby-2-4-0-released.md

```diff
@@ -69,12 +69,12 @@ Rubyにはスレッド待ちのデッドロックを検出する機能があり
 * ext/tk is now removed from stdlib [Feature #8539](https://bugs.ruby-lang.org/issues/8539)
 * XMLRPC is now removed from stdlib [Feature #12160](https://bugs.ruby-lang.org/issues/12160)
 
-See [NEWS](https://github.com/ruby/ruby/blob/v2_4_0_preview3/NEWS)
-and [ChangeLog](https://github.com/ruby/ruby/blob/v2_4_0_preview3/ChangeLog)
+See [NEWS](https://github.com/ruby/ruby/blob/v2_4_0/NEWS)
+and [ChangeLog](https://github.com/ruby/ruby/blob/v2_4_0/doc/ChangeLog-2.4.0)
```

Comment:
@kachick @nurse

`en` version simply refers to the "commit logs" instead of ChangeLog.